package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "stylelint-config-bootstrap",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Stylelint configuration for Bootstrap",
+  "main": "css/index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/twbs/stylelint-config-bootstrap.git"
+  },
+  "keywords": [
+    "bootstrap",
+    "stylelint",
+    "config",
+    "lint"
+  ],
+  "author": "The Bootstrap Authors (https://github.com/twbs/stylelint-config-bootstrap/graphs/contributors)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/twbs/stylelint-config-bootstrap/issues"
+  },
+  "homepage": "https://github.com/twbs/stylelint-config-bootstrap#readme",
+  "peerDependencies": {
+    "stylelint": "^9.9.0"
+  },
+  "dependencies": {
+    "stylelint-config-recommended-scss": "^3.2.0",
+    "stylelint-config-standard": "^18.2.0"
+  },
+  "files": [
+    "css/*.js",
+    "scss/*.js"
```

Comment:
@XhmikosR , I think we can just link to the `index.js` files here?

Reply:
Yeah, either way works, whatever you prefer. I wasn't sure if you planned to add more files later to the repo.